package.json

```diff
@@ -70,16 +70,17 @@
 		"@changesets/cli": "^2.27.11",
 		"@commitlint/cli": "^19.6.1",
 		"@commitlint/config-conventional": "^19.6.0",
+		"@csstools/postcss-bundler": "^2.0.6",
```

Comment:
Added the postcss-bundler for the Storybook styles being processed so we have a more optimized and usable product for even our internal CSS.